samples/openapi3/client/petstore/typescript/builds/object_params/PetApi.md

```diff
@@ -215,9 +211,7 @@ const apiInstance = new petstore.PetApi(configuration);
 
 let body:petstore.PetApiFindPetsByTagsRequest = {
   // Array<string> | Tags to filter by
-  tags: [
-    "tags_example",
-  ],
+  tags: [tags_example],
```

Comment:
Looks like the change is breaking this.

Reply:
@therve good catch! 🙇  I think I have found a solution.